ibridgesgui/gui_utils.py

```diff
@@ -74,6 +74,26 @@ def prep_session_for_copy(session, error_label) -> pathlib.Path:
     return None
 
 
+def combine_diffs(session, sources: list, destination: Union[IrodsPath, pathlib.Path]) -> dict:
```

Comment:
This function should probably be made obsolete by implementing it in the ibridges library, but for now a useful feature!

Reply:
Be my guest ;)